.gitignore

```diff
@@ -50,6 +50,7 @@ DerivedData
 ZipBuilder/Packages
 ZipBuilder/*.xcodeproj
 ZipBuilder/Package.resolved
+ZipBuilder/.swiftpm
```

Comment:
What is this file for?

Reply:
This folder was created by Swift PM. According to the [thread](https://forums.swift.org/t/files-created-when-opening-spm-package-in-xcode-11/28589) it may contain things that should be checked in, but I didn't find anything useful there so far. We can revisit ignoring it later.